### PR TITLE
feat(jest): 35% coverage for store/accounts/actions

### PR DESCRIPTION
**Pull request by drepram**
**What this PR does** 📖

Self-explanatory PR, but am putting this ticket into the backlog because of the current issue with the .hash function.

**Which issue(s) this PR fixes** 🔨

AP-661 (partially)

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

File: store/accounts/actions.test.ts [hunk 1]
```diff
@@ -0,0 +1,436 @@
...
+      pin: '?{fDn4s8I5~sb@*F858{]CZ@A',
+    }
+
+    const commit = jest.fn()
+    // CPrototype.hash = jest.fn()
+    // CPrototype.hash.mockReturnValueOnce(
+    //   'fddd5093b61663c64c309a0352b3762e4f6b7277b1ec7f2ac64f4b696c66ab91',
+    // )
+    // CPrototype.decryptWithPassword = jest.fn()
+    // CPrototype.decryptWithPassword.mockReturnValueOnce('decrypted phrase')
```
Comment (stavares843) [inline store/accounts/actions.test.ts:140]: do we need this? 🔨

**Review by phillsatellite — APPROVED** [review]

**Review by stavares843 — APPROVED** [review]

**Pull request merged**